### PR TITLE
Add support for gunicorn in Dashboard

### PR DESCRIPTION
**Pull request by sevein**

### CODE REVIEW

File: src/dashboard/src/wsgi.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import os
+
+from django.core.wsgi import get_wsgi_application
+
+
+application = get_wsgi_application()
```
Comment (Hwesta) [inline src/dashboard/src/wsgi.py:6]: This is intentionally not reusing `src/dashboard/src/apache/django.wsgi`?

File: src/dashboard/src/requirements/base.txt [hunk 2]
```diff
@@ -9,6 +9,7 @@ django-autoslug==1.7.1
 django-annoying==0.7.7
 elasticsearch>=1.0.0,<2.0.0
 gearman==2.0.2
+gunicorn==19.4.5
```
Comment (Hwesta) [inline src/dashboard/src/requirements/base.txt:12]: If this is in base, remove from local.txt please.

Comment (sevein) [inline src/dashboard/src/requirements/base.txt:12]: Thank you :) - done now.

File: src/dashboard/src/wsgi.py (hunk 1 above)
Comment (sevein) [inline src/dashboard/src/wsgi.py:6]: Yes, that's being ignored purposely. There might be more changes after testing (like finding a place for ES initialization when using gunicorn), but this is our first barrier.

**Comment by Hwesta** [comment]
:+1:

**Pull request merged**